WooCommerce/Classes/AppDelegate.swift

```diff
@@ -276,6 +281,15 @@ private extension AppDelegate {
         appRating.systemWideSignificantEventCountRequiredForPrompt = WooConstants.systemEventCount
         appRating.setVersion(version)
     }
+
+    /// Set up Wormholy only in Debug build configuration
+    ///
+    func setupWormholy() {
+        #if DEBUG
```

Comment:
for `#if DEBUG` in code, maybe we can use `BuildConfiguration.current == .localDeveloper` like in `FeatureFlag` https://github.com/woocommerce/woocommerce-ios/blob/develop/WooCommerce/Classes/System/DefaultFeatureFlagService.swift#L5?

Reply:
This sounds good, but I think that if I use `if BuildConfiguration.current == .localDeveloper` everything inside the block will be compiled, so the build will fail if the configuration is different from Debug, since Wormholy will be integrated via Pod only in Debug configuration.

Reply:
ooh I see! `#if DEBUG` looks good then 👍